packages/itwinui-css/src/menu/classes.scss

```diff
@@ -4,6 +4,10 @@
 
 .iui-menu {
   @include iui-list-menu;
+
+  .iui-listbox-wrapper & {
```

Comment:
Thank you @elephantcatdog for the tip as to what was failing.  This change fixed the failed test images:
```diff
- &:where([role='listbox'])
+ .iui-listbox-wrapper & {
```

But depending on how #843 pans out, this may not be needed.  @mayank99